clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/op/job/KubectlJobExecutor.java

```diff
@@ -453,8 +453,14 @@ public ImmutableList<KubernetesManifest> list(
             parseManifestList());
 
     if (status.getResult() != JobResult.Result.SUCCESS) {
-      throw new KubectlException(
-          "Failed to read " + kinds + " from " + namespace + ": " + status.getError());
+      boolean permissionError =
```

Comment:
Is using Apache Commons necessary? The next if block below just does a simple contains